cardano-testnet/src/Parsers/Run.hs

```diff
@@ -65,8 +65,10 @@ createEnvOptions CardanoTestnetCreateEnvOptions
   , createEnvGenesisOptions=genesisOptions
   , createEnvOutputDir=outputDir
   } =
-    void $ H.check $ testnetRoutine (UserProvidedEnv outputDir) $ createTestnetEnv
-      testnetOptions genesisOptions
+    void $ H.check $ testnetRoutine (UserProvidedEnv outputDir) $
+      createTestnetEnv
+        testnetOptions genesisOptions
+        NoUserProvidedData NoUserProvidedData NoUserProvidedData
```

Comment:
We should put a comment here stating that the parsers not providing a way to supply the genesis values is intentional.